### ISSUE TITLE
unknown: mosca in in Node-RED on IBM Cloud

### ISSUE THREAD

**Issue by gitjps** [zuhito/node-red-contrib-mqtt-broker#20]
I followed the instructions on http://www.steves-internet-guide.com/install-mosca-mqtt-broker-node-red/ on IBM Cloud.

I installed the node-red-contrib-mqtt-broker, but got the error "unknown: mosca in". 

See https://stackoverflow.com/questions/53829942/unknown-mosca-in-in-node-red-on-ibm-cloud

**Comment by martin-doyle** [comment]
Obviously, the installation of the node did not succeed. I do not have any experience with the installation via Node-Red Admin. If you run Node-Red on Windows please check [Installing with errors on Windows10](https://github.com/zuhito/node-red-contrib-mqtt-broker/issues/18). On Linux try installing the node globally with `sudo npm install -g --unsafe-perm node-red-contrib-mqtt-broker`.
